packages/esm-appointments-app/src/admin/appointment-services/appointment-services.component.tsx

```diff
@@ -57,94 +57,104 @@ const AppointmentServices: React.FC<AppointmentServicesProps> = () => {
         return (
           <Form onSubmit={props.handleSubmit} className={styles.appointmentServiceContainer}>
             <p className={styles.formTitle}>{t('createAppointmentService', 'Create appointment service')}</p>
-            <TextInput
-              light
-              id="name"
-              invalidText={t(props.errors.name)}
-              labelText={t('appointmentServiceName', 'Appointment service name')}
-              placeholder={t('appointmentServiceName', 'Appointment service name')}
-              invalid={!!(props.touched && props.errors.name)}
-              onChange={props.handleChange}
-              value={props.values.name}
-              name="name"
-              onBlue={props.handleBlur}
-            />
-            <TimePicker
-              light
-              className={styles.timePickerInput}
-              invalid={!!(props.touched && props.errors.startTime)}
-              pattern="([\d]+:[\d]{2})"
-              name="startTime"
-              value={props.values.startTime}
-              onChange={props.handleChange}
-              labelText={t('startTime', 'Start Time')}
-              id="start-time-picker">
-              <TimePickerSelect
-                name="startTimeTimeFormat"
+            <Layer>
+              <TextInput
```

Comment:
Is this relevant to this PR only?

Reply:
As I explained on the call, yes. This clears warnings about the soon-to-be-deprecated `light` prop for Carbon components when running tests.